### PR TITLE
sass'ing skeleton

### PR DESCRIPTION
**Pull request by jgtoriginal**
Hi there, it's been a long while that I don't use css any more.
I tried with less and then with sass and never went back again.
Last week I've been making some research on css frameworks for kicking off a new project where I work.
And I found skeleton, that is not only what I was after, but also way better than many others.
The only downside for me, was it not being in sass.
But that wasn't a real issue, cause I could make that happen straight away.
So here it is my pull request. My sharing with the community.
Thx.

jgtoriginal.

### CODE REVIEW

**Comment by johnny-spindrift** [comment]
so what was done jgtoriginal?
did you just combine the files? (which is fine, I just wondered if you added any other sass goodness)

**Comment by emigre** [comment]
It seems that @dhg is adding sass in 2.0: 1c90b35404ec3543ccbc012d85f2eb3b8ee39b92

**Comment by dhg** [comment]
Hey all - so sorry about never getting to this in a timely fashion. I left Skeleton in disrepair and without an active developer for far too long. I'm actively developing it again and committed to sticking with it this time around. 

This conversation is happening again now, but here is the primary thread for that conversation: https://github.com/dhg/Skeleton/pull/194

Tldr; not going to be in core of Skeleton, but will be other authors who create repos that I will review!
